routes/activities/create.js

```diff
@@ -63,7 +63,17 @@ const handleCreate = async (req, res, reader) => {
       break
 
     case 'Note':
-      const resultNote = await Note.createNote(reader, body.object)
+      const noteObj = {
+        noteType: 'Note',
+        content: body.object.content,
+        selector: body.object.selector,
+        json: body.object.json,
+        context: body.object.context,
+        inReplyTo: body.object.inReplyTo,
+        noteType: body.object.noteType
+      }
```

Comment:
the only change you do here to the noteObj is add a noteType. That should be send as part of the request (I might have forgotten it in my tests). You can stick with the body.object